pkg/scheduler/scheduler.go

```diff
@@ -104,22 +104,22 @@ func (s *Scheduler) schedule(ctx context.Context) {
 	usedCohorts := sets.NewString()
 	for i := range entries {
 		e := &entries[i]
-		if !e.admissible {
+		if e.inadmissible {
 			continue
 		}
 		c := snapshot.ClusterQueues[e.ClusterQueue]
 		if len(e.borrows) > 0 && c.Cohort != nil && usedCohorts.Has(c.Cohort.Name) {
-			e.admissible = false
+			e.inadmissible = true
```

Comment:
There is a bit of inconsistency here.
We mark the workloads with ("cohort used in this cycle") as not inadmissible when re-enqueue.

Reply:
good catch, I guess we can use nominated, I hesitated initially because it didn't make sense when reversing this flag in case admit fails (few lines below)

Reply:
actually we can have both flags, and that will solve the problem

Reply:
In that case, we can have `inadmissible` and `nominated`, and get rid of `skipped`?

Reply:
I changed to enum, take a look :)

Reply:
Two inadmissible and nominated flags would look nice too, i am ok with either, wdyt?